sample/src/screens/PerformanceTestScreen.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+
+/**
+ * A screen to test performance timing.
```

Comment:
Clarify why this is here: `global.performance.now();` can return far away in the future values so this code compares `global.performance.now();` with the actual value to display any discrepancies